src/components/Form/Form.js

```diff
@@ -4,32 +4,65 @@ import {useRouter} from 'next/router';
 import styled from 'styled-components';
 
 import useStore from '../../Hooks/useStore';
+import {pxToRem} from '../../utils/unit';
 import StyledButton from '../Button/StyledButton';
 import Headline2 from '../Headline/Headline2';
-
 export default function Form() {
 	const addEntry = useStore(state => state.addEntry);
 	const router = useRouter();
-
 	function handleSubmit(event) {
 		event.preventDefault();
-
 		const formData = new FormData(event.target);
-		const {firstEntry, secondEntry, thirdEntry} = Object.fromEntries(formData);
-
+		const {firstEntry, secondEntry, thirdEntry, radio} = Object.fromEntries(formData);
 		event.target.reset();
-		const entry = {id: nanoid(), first: firstEntry, second: secondEntry, third: thirdEntry};
+		const entry = {
+			id: nanoid(),
+			first: firstEntry,
+			second: secondEntry,
+			third: thirdEntry,
+			radio1: radio,
+			radio2: radio,
+			radio3: radio,
+			radio4: radio,
+			radio5: radio,
+			radio6: radio,
```

Comment:
I think you don't need radio2 to radio6, because you just store the same value in each of the keys. Later in diary.js you only work with radio1. Also I would give the keys a more specific name, for example "mood".